torch/csrc/jit/runtime/register_backend_ops.cpp

```diff
@@ -0,0 +1,87 @@
+#include <ATen/core/stack.h>
+#include <torch/csrc/jit/backends/backend.h>
+#include <torch/csrc/jit/runtime/custom_operator.h>
+#include <torch/custom_class_detail.h>
+
+namespace torch {
+namespace jit {
+namespace {
+c10::AliasAnalysisKind aliasAnalysisFromSchema() {
+  return c10::AliasAnalysisKind::FROM_SCHEMA;
+}
+
+c10::AliasAnalysisKind aliasAnalysisSpecialCase() {
+  return c10::AliasAnalysisKind::INTERNAL_SPECIAL_CASE;
+}
+
+// Register operators that allow for calling into the custom compile and execute
+// methods for a JIT backend.
+RegisterOperators reg({
+    Operator(
+        "prim::backend_create(str a, Any a) -> Capsule",
```

Comment:
Both parameters have the same name here, might mess stuff up for kwarg schema matching

I'd suggest giving these parameters meaningful names as well